room/room-compiler-processing/src/main/java/androidx/room/compiler/processing/javac/JavacBasicAnnotationProcessor.kt

```diff
@@ -32,15 +32,18 @@ import javax.lang.model.element.Element
 abstract class JavacBasicAnnotationProcessor :
     BasicAnnotationProcessor(), XBasicAnnotationProcessor {
 
-    // This state is cached here so that it can be shared by all steps in a given processing round.
-    // The state is initialized at beginning of each round using the InitializingStep, and
-    // the state is cleared at the end of each round in BasicAnnotationProcessor#postRound()
-    private var cachedXEnv: JavacProcessingEnv? = null
+    private val xEnv: JavacProcessingEnv by lazy { JavacProcessingEnv(processingEnv) }
+
+    final override val xProcessingEnv: XProcessingEnv by lazy { xEnv }
```

Comment:
Can be a getter-only property without a lazy backing field:
`final override val xProcessingEnv: XProcessingEnv get() = xEnv`

Reply:
Done.